openeo/rest/connection.py

```diff
@@ -575,7 +589,8 @@ def list_services(self) -> dict:
         :return: data_dict: Dict All available service types
         """
         # TODO return parsed service objects
-        return self.get('/services').json()
+        data = self.get('/services').json()
+        return data
```

Comment:
is this preparation, or you forgot VisualList?

Reply:
Could be preparation for the future, but to be honest, it was just a too greedy me copy/pasting things... reverted.